module/vision/LUTClassifier/src/GoalEnhancer.cpp

```diff
@@ -87,6 +89,7 @@ namespace vision {
          */
 
         Line horizon(convert<double, 2>(classifiedImage.horizon.normal), classifiedImage.horizon.distance);
+        // Plane<3> horizon;
```

Comment:
Should use the real horizon for this mini ransac through centre points, otherwise it'll end up classifying a whole lot of junk with very dense classification lines.